frontend/components/Permissions/Permissions.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { HTMLProps, useState } from "react";
+
+import { FormElementProps } from "nhsuk-react-components/lib/util/types/FormTypes";
+import styled from "styled-components";
+
+import { ChevronBottomIcon, ChevronTopIcon } from "../Icon";
+import PermissionsRadio from "../PermissionsRadio/PermissionsRadio";
+
+interface PermissionsProps
+  extends HTMLProps<HTMLInputElement>,
+    FormElementProps {
+  inputRef?: (inputRef: HTMLInputElement | null) => any;
+}
+
+const TitleContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 24px;
+
+  h3 {
+    margin: 0;
+  }
+
+  button {
+    padding: 0 0 0 20px;
```

Comment:
```suggestion
    margin: 0 0 0 20px;
```
Suggest changing this to a margin so that the focus state bg-color suggested below isn't affected by the padding